chia/full_node/fee_estimate2.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from typing import Optional, List
+from chia.util.ints import uint32
+from chia.util.streamable import Streamable, streamable
+
+
+@dataclass(frozen=True)
+@streamable
+class FeeEstimate(Streamable):
+    """
+    Estimates here will be x mojo / 1 clvm_cost.
+    Negative value indicates error.
+    """
+
+    error: Optional[str]
+    short: str
+    medium: str
+    long: str
+
+
+@dataclass(frozen=True)
+@streamable
+class FeeStatBackup(Streamable):
+    type: str
```

Comment:
what is the meaning of type here ?

Reply:
Whoa, that file should not have been committed. Thanks Almog!